scripts/xtask

```diff
@@ -5,5 +5,5 @@ readonly SCRIPTS_DIR="$(dirname "$0")"
 source "$SCRIPTS_DIR/common"
```

Comment:
It looks like permissions for this script changed somehow, it seems not to work any more in CI.

Reply:
Checked the permissions for the file they haven't changed, don't know why CI is failing

Reply:
GitHub shows `100755 → 100644` near this file name which *I think* means the permissions have changed, somehow?

Reply:
Hey @tiziano88, Figured out the issue ME using NTFS with windows is causing the issue with the permissions. I will fix that  and push with changes

Reply:
Hey @tiziano88 I have pushed the changes, do check it out